### ISSUE TITLE
Python app installation fails in PyCharm

### ISSUE THREAD

**Issue by brikelly** [getappmap/appmap-js#1308]
The console output looks messed up and garbled somewhat. PyCharm CE 2023.2, latest AppMap plugin (0.44).
See https://www.loom.com/share/bd20cf55d4c243db934ffd0ee031200a?sid=e0630fa6-30fa-42f3-b630-81e41c58af5e

Joachim looked at this and determined it's an issue in the installer code, not the JetBrains plugin.

**Comment by kgilpin** [comment]
This seems like a duplicate or regression of an issue that was reported
months ago and supposedly fixed once.

The installer code hasn’t changed. It works in the VSCode terminal. It’s
hard for me to believe that it’s due to any change in the installer.

**Comment by brikelly** [comment]
Yes there was a similar issue, will find that and link to it. It might be that the original change works only for VS Code but not for PyCharm, for whatever reason.

**Comment by kgilpin** [comment]
It was not a VSCode issue.

**Comment by brikelly** [comment]
I didn't say it was. I'm saying that there may be something specific to PyCharm that makes it fail there.

**Comment by jansorg** [comment]
my observation was posted at https://github.com/getappmap/appmap-intellij-plugin/issues/346#issuecomment-1665447995.

@brikelly Could you please open a terminal window (outside of PyCharm) and manually execute the command shown in the first line?
That's the failed command shown in your loom. If it completes successfully, then it's very likely an issue with the IntelliJ plugin, if it fails then I don't think it's a problem with the plugin. 

![image](https://github.com/getappmap/appmap-js/assets/11473063/1b532a1c-bc4c-4e1b-9434-20bde68440be)

**Comment by brikelly** [comment]
Thanks @jansorg, it failed the same way in iTerm as it did within PyCharm.

**Comment by brikelly** [comment]
This fails now in both PyCharm and VS Code.

**Comment by kgilpin** [comment]
What are the details here? Are all the Python users failing?

**Comment by brikelly** [comment]
Thankfully nobody else has been able to reproduce the same issue as me so far. Looks like an issue in my environment with PyEnv. Disabling PyEnv fixes the issue for me, at least on VS Code.
